erlang-25.yaml

```diff
@@ -0,0 +1,73 @@
+package:
+  name: erlang-25
+  version: 25.3.2.2
+  epoch: 0
+  description: General-purpose programming language and runtime environment
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      # mnesia depends on the ca-certificates bundle
+      - ca-certificates-bundle
+    provides:
+      - erlang=25.3.2.2
```

Comment:
does this conflict with the erlang-26 provides?

Reply:
This is how we do it with go: https://github.com/wolfi-dev/os/blob/main/go-1.19.yaml#L10